packit_service/worker/build/koji_build.py

```diff
@@ -113,6 +113,17 @@ def supported_koji_targets(self):
         return self._supported_koji_targets
 
     def run_koji_build(self) -> TaskResults:
+        if not self.is_scratch:
+            msg = (
+                "It is not possible to run non-scratch production builds from upstream."
```

Comment:
I would try to make it shorter. The space is really limited on Github so I would try to put most important info to the beginning...

```suggestion
                "Non-scratch builds not possible from upstream."
```

Reply:
right, done